src/main/kotlin/com/example/noswimmingserver/domain/teacher/presentation/TeacherController.kt

```diff
@@ -1,22 +1,29 @@
 package com.example.noswimmingserver.domain.teacher.presentation
 
-import com.example.noswimmingserver.domain.teacher.presentation.dto.response.AccountResponse
+import com.example.noswimmingserver.domain.teacher.presentation.dto.request.UpdateTeacherInfoRequest
+import com.example.noswimmingserver.domain.teacher.presentation.dto.response.TeacherAccountResponse
 import com.example.noswimmingserver.domain.teacher.service.CreateRandomAccountService
+import com.example.noswimmingserver.domain.teacher.service.UpdateTeacherInfoService
 import org.springframework.http.HttpStatus
-import org.springframework.web.bind.annotation.PostMapping
-import org.springframework.web.bind.annotation.RequestMapping
-import org.springframework.web.bind.annotation.ResponseStatus
-import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.bind.annotation.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)